cns/restserver/internalapi.go

```diff
@@ -253,7 +253,8 @@ func (service *HTTPRestService) syncHostNCVersion(ctx context.Context, channelMo
 		}
 		if localNCVersion > nmaNCVersion {
 			// This means that NMAgent reported a regression in the NC Version.
-			panic(fmt.Errorf("NC version from NMA is decreasing: have %d, got %d", localNCVersion, nmaNCVersion))
+			err := fmt.Sprintf("NC version from NMA is decreasing: have %d, got %d", localNCVersion, nmaNCVersion)
+			panic("NC Version Regression: " + err)
```

Comment:
this is not an error and shouldn't be called `err` since that's misleading. but since it's just a string, can we inline it?

```suggestion
			panic(fmt.Sprintf("NC version from NMA has decreased from previously recorded: have %d, got %d", localNCVersion, nmaNCVersion))
```

Reply:
I had initially added this inline. But golint complained. I will update the variable name.